Jenkinsfile

```
@@ -138,8 +138,9 @@ def cmake_build(Map conf=[:]){
     sh cmd
 
     // Only archive from master or develop
-    if (package_build == true && (env.BRANCH_NAME == "develop" || env.BRANCH_NAME == "master")) {
+    if (package_build == true && (env.BRANCH_NAME == "develop" || env.BRANCH_NAME == "master" || env.BRANCH_NAME == env.MIOPEN_PERF_BRANCH)) {
```

[Q] Where is `MIOPEN_PERF_BRANCH` defined? In the Jenkins' web UI?

[Q] Is it supposed to be manually managed?

[Recommendation] Rename to
```suggestion
    if (package_build == true && (env.BRANCH_NAME == "develop" || env.BRANCH_NAME == "master" || env.BRANCH_NAME == env.MIOPEN_GOLDEN_PERF_BRANCH)) {
```

---

this is defined once in the Jenkins configurations and the branch is managed just like our develop branch etc

---

I see. Managed manually. Ok. Resolved.